api/v1beta1/recording_types.go

```diff
@@ -112,6 +112,7 @@ type RecordingStatus struct {
 // +kubebuilder:resource:path=recordings,scope=Namespaced
 
 // Recording is the Schema for the recordings API
+//+operator-sdk:csv:customresourcedefinitions:resources={{Pod,v1beta1},{Secret,v1},{Service,v1}}
```

Comment:
Pod should be `v1`.

Reply:
Whoops, I had that as `Recording` initially as we discussed on irc. Glad you caught it.